web/src/containers/AriaAnnounce.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+
+import { getAriaAnnouncement } from '../reducers/aria';
+
+const AriaAnnounce = (props) =>{
+    // const [ariaMessage, getAriaAnnouncement] = useState(null);
```

Comment:
I was tryin' to be clever but confused myself. Figured I could go back to this after I figured it out the old way. 😅